test/Desktop/test_desktop.py

```diff
@@ -1,19 +1,17 @@
+from selenium.common.exceptions import WebDriverException
```

Comment:
By default this file is hidden due to a large amount of changes, unlike the doc pages though this should be reviewed as they are all manual changes I made.